ksql-common/src/main/java/io/confluent/ksql/util/KsqlConstants.java

```diff
@@ -25,6 +25,9 @@
       "WINDOW_CHANGE_LOG_ADDITIONAL_RETENTION";
   public static final String SINK_TIMESTAMP_COLUMN_NAME = "TIMESTAMP";
 
+  public static final String SCHEMA_REGISTRY_VALUE_SUFFIX = "-value";
+  public static final String AVRO_SCHEMA_ID = "SCHEMA_ID";
```

Comment:
Shouldn't the string literal also be `AVRO_SCHEMA_ID`. Otherwise it would look quite ambiguous when reading a .properties file.

Reply:
Good point, I'll change it to `AVRO_SCHEMA_ID`.